hybrasyl/Inventory.cs

```diff
@@ -696,7 +696,7 @@ public byte[] SlotOf(string name)
             var slotsByte = new byte[slotsInt.Length];
             for(int i = 0; i < slotsInt.Length; i++)
             {
-                slotsByte[i] += 1;
+                slotsByte[i] = (byte)(slotsInt[i] + 1);

```

Comment:
In my trials, this resulted in slotsByte[] values all being 1. This resulted in an error if I tried to remove an item from the second slot.

Reply:
interesting, i just noticed that it's actually a bug because all values would be 0 in that array. good work. (had to look at it in visual studio to notice it fully)